test/jest/persistence.spec.ts

```diff
@@ -0,0 +1,145 @@
+import Store, { IStore } from '../../src/store';
+import { IMParticleWebSDKInstance } from '../../src/mp-instance';
+import { SDKInitConfig } from '../../src/sdkRuntimeModels';
+
+describe('Persistence store', () => {
```

Comment:
Nit: `store` is redundant here.

```suggestion
describe('Persistence', () => {
```